mne_bids/path.py

```diff
@@ -599,8 +599,7 @@ def match(self):
 
         fnames = self.root.rglob(search_str)
         # Only keep files (not directories), and omit the JSON sidecars.
-        fnames = [f.name for f in fnames
-                  if f.is_file() and f.suffix != '.json']
+        fnames = [f.name for f in fnames if f.is_file()]
```

Comment:
We made a conscious decision to exclude JSON files here – I don't remember the reason though. But this needs to be discussed. Also the comment doesn't make sense now anymore

Reply:
I couldn't find any explicit reason from the PR awhile back: https://github.com/mne-tools/mne-bids/pull/507/

The only reason I can see for excluding json is if you want a list of the actual raw EEG/MEg/ieeg datasets, rather then their sidecars too?

Reply:
Any major reason to exclude jsons? Or should I add a kwarg to `match(include_json=False)`?